src/find/azext_find/__init__.py

```diff
@@ -29,7 +37,7 @@ def load_command_table(self, _):
 
     def load_arguments(self, _):
         with self.argument_context('find') as c:
-            c.positional('question', help='Questions about Azure CLI commands.')
+            c.positional('cli_command', help='A CLI command for which you need an example.')
```

Comment:
The input does not need to be a command right? It can be a query, group or command.

Reply:
Yes, it doesn't need to be a command. But, I thought question is also misleading. Do you have any other suggestions?

Reply:
I think `query` is more encompassing of all three as a query can be a command.

Reply:
query is a global argument, I don't think we can use that.

Reply:
I have it as cli_term now, do you want me to change it to cli_query?